docs/application/native/api/iot-headless/6.0/group__CAPI__MEDIA__AUDIO__OUT__MODULE.html

```diff
@@ -0,0 +1,1095 @@
+    <tr><td class="paramname"><a class="el" href="group__CAPI__MEDIA__AUDIO__IO__MODULE.html#ggac89b56354793b3ba70c79a9716189e7da3e528888c4acd619f4c54996c58a72e1">AUDIO_IO_ERROR_INVALID_PARAMETER</a></td><td>Invalid parameter</td></tr>
+  </table>
+  </dd>
+</dl>
+<dl class="see"><dt><b>See also:</b></dt><dd><a class="el" href="group__CAPI__MEDIA__AUDIO__OUT__MODULE.html#ga13fec0c80d91d5b62d0184dffc49d946" title="Unregisters the state changed callback function of the audio output handle.">audio_out_unset_state_changed_cb()</a> </dd></dl>
+
+</div>
+</div>
+<a class="anchor" id="gad6f03c7d42778edb5382ab0bcf65ff1c"></a><!-- doxytag: member="audio_io.h::audio_out_set_stream_cb" ref="gad6f03c7d42778edb5382ab0bcf65ff1c" args="(audio_out_h output, audio_out_stream_cb callback, void *user_data)" -->
+<div class="memitem">
+<div class="memproto">
+      <table class="memname">
+        <tr>
+          <td class="memname">int <a class="el" href="group__CAPI__MEDIA__AUDIO__OUT__MODULE.html#gad6f03c7d42778edb5382ab0bcf65ff1c">audio_out_set_stream_cb</a> </td>
+          <td>(</td>
+          <td class="paramtype"><a class="el" href="group__CAPI__MEDIA__AUDIO__OUT__MODULE.html#ga579e22d913af7acb1ca2e2d97a8c755c">audio_out_h</a>&#160;</td>
+          <td class="paramname"><em>output</em>, </td>
+        </tr>
+        <tr>
+          <td class="paramkey"></td>
+          <td></td>
+          <td class="paramtype"><a class="el" href="group__CAPI__MEDIA__AUDIO__OUT__MODULE.html#ga6d20cceb7461a5ff5706b6983a93dd81">audio_out_stream_cb</a>&#160;</td>
+          <td class="paramname"><em>callback</em>, </td>
+        </tr>
+        <tr>
+          <td class="paramkey"></td>
+          <td></td>
+          <td class="paramtype">void *&#160;</td>
+          <td class="paramname"><em>user_data</em>&#160;</td>
+        </tr>
+        <tr>
+          <td></td>
+          <td>)</td>
+          <td></td><td></td>
+        </tr>
+      </table>
+</div>
+<div class="memdoc">
+
+<p>Sets an asynchronous (event) callback function to handle playing PCM (pulse-code modulation) data. </p>
+<p><em>callback</em> will be called when you can write a PCM data. It might cause dead lock if change the state of audio handle in callback. (ex: <a class="el" href="group__CAPI__MEDIA__AUDIO__OUT__MODULE.html#ga5bdb945e25a68be9360fb8cc6d79d5f4" title="Releases the audio output handle, along with all its resources.">audio_out_destroy()</a>, <a class="el" href="group__CAPI__MEDIA__AUDIO__OUT__MODULE.html#ga5670195c69022860d9365c32ac7c872f" title="Prepares the audio output for playback, this must be called before audio_out_write().">audio_out_prepare()</a>, <a class="el" href="group__CAPI__MEDIA__AUDIO__OUT__MODULE.html#ga73e4c12e1e642717063d257046ff850f" title="Unprepares the audio output.">audio_out_unprepare()</a>) Recommend to use as a VOIP only. Recommend not to hold callback too long.(it affects latency)</p>
```

Comment:
```suggestion
<p><em>callback</em> will be called when you can write the PCM data. It might cause dead lock if change the state of audio handle in callback. (ex: <a class="el" href="group__CAPI__MEDIA__AUDIO__OUT__MODULE.html#ga5bdb945e25a68be9360fb8cc6d79d5f4" title="Releases the audio output handle, along with all its resources.">audio_out_destroy()</a>, <a class="el" href="group__CAPI__MEDIA__AUDIO__OUT__MODULE.html#ga5670195c69022860d9365c32ac7c872f" title="Prepares the audio output for playback, this must be called before audio_out_write().">audio_out_prepare()</a>, <a class="el" href="group__CAPI__MEDIA__AUDIO__OUT__MODULE.html#ga73e4c12e1e642717063d257046ff850f" title="Unprepares the audio output.">audio_out_unprepare()</a>) It is recommended to use as a VOIP only, and not to hold the callback for too long.(it affects latency)</p>

Reply:
> @nawab-samsung Hello, nawab
> This change is for adding missing modules - Audio IO / Wav Player.
> and, need to change header files, instead of modifying html directly.
> 
> To apply for your recommendation, need to do below scenario.
> 
> 1. Each developers need to fix their header files (.h) and submit them to gerrit server and need to get approval from PM part
>    After that, header files can be applied to image file.
>    Further, need to consider fixing other version (4.0/5.5/6.0/6.5) and other header files,
>    e.g. wav can be changed to WAV for this PR cases. but, other headers also may have "wav".
>    If they are also changed to "WAV", it means many files should be fixed also.
> 2. When new image file including new header files is released,
>    I start to generate API Reference. and, can update PR.
> 
> In case of 1), it takes very long time... so, I am using this page for better API Reference.
> https://code.sec.samsung.net/confluence/pages/viewpage.action?pageId=194846376
> 
> Would you discuss this PR with @mijin85cho ?
> And, let me know MUST-FIX items. I will find correct header files and owners and add it to issues pages upper link.

@sgchoi5, thanks for explaining the concerns. As per me, there are no must-fix comment that needs to be addressed, so we can approve this PR as is.